pkg/controllers/nodeclaim/tagging/controller.go

```diff
@@ -0,0 +1,179 @@
+/*
```

Comment:
Can we create a `suite_test.go` for this controller. Looking at `pkg/controllers/nodeclass/suite_test.go` might be a good start to look at what the testing should look like and how the mocks can get setup